src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
+Main-Class: net.justonedev.kit.Main
+
```

Comment:
Manifests should be generated by maven

Reply:
Done, sorry about that. Result of module merging.